packages/icons/src/index.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import * as icons from "./index";
+import * as legacyIcons from "./legacy";
+
+export const Icons = () => {
+  return (
+    <>
+      <style>{`
+        svg {width: 100%; height: 100%;}
+    `}</style>
+      <div style={{ display: "flex", flexWrap: "wrap" }}>
+        {Object.entries(icons).map(
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          ([name, Icon]: [string, any]) => {
+            if (name.endsWith("Icon") === false) {
+              return null;
+            }
+            return (
+              <div
+                key={name}
+                style={{
+                  width: 150,
+                  height: 100,
+                  margin: 5,
+                  padding: 5,
+                  border: "solid 1px #f5f5f5",
+                  display: "flex",
+                  flexDirection: "column",
+                  alignItems: "center",
+                  justifyContent: "center",
+                }}
+              >
+                <div
+                  style={{
+                    display: "flex",
+                    width: 32,
+                    height: 32,
+                    alignItems: "center",
+                    justifyContent: "center",
+                    marginBottom: 16,
+                  }}
+                >
+                  <Icon />
```

Comment:
just looked again, why didn't you just pass width and hight prop to the icon instead of <style>?

Reply:
Didn't realise it's supported 🤦